pkg/frameworks/framework.go

```diff
@@ -0,0 +1,40 @@
+package frameworks
+
+import (
+	"fmt"
+
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha1"
+	"github.com/kubeflow/kfserving/pkg/frameworks/custom"
+	"github.com/kubeflow/kfserving/pkg/frameworks/tensorflow"
+	v1 "k8s.io/api/core/v1"
+)
+
+type FrameworkHandler interface {
+	CreateModelServingContainer(modelName string) *v1.Container
+	ValidateSpec() error
+}
+
+func Get(modelSpec *v1alpha1.ModelSpec) (interface{ FrameworkHandler }, error) {
+	handlers := []interface{ FrameworkHandler }{}
+	if modelSpec.Custom != nil {
+		handlers = append(handlers, &custom.CustomFramework{Spec: modelSpec.Custom})
+	}
+	if modelSpec.XGBoost != nil {
+		// TODO: add fwk for xgboost
+		handlers = append(handlers, &custom.CustomFramework{Spec: modelSpec.Custom})
```

Comment:
Perhaps leave these lines out to avoid confusion. Empty if statements are fine in go, right?

Reply:
removed code, left TODO